swan-lake/learn/security/authentication-and-authorization.md

```diff
@@ -537,141 +316,105 @@ curl -k -v https://localhost:9091/hello -H 'Authorization: Bearer <token>'
         keyStore: {
-            path: config:getAsString("b7a.home") + "/bre/security/ballerinaKeystore.p12",
+            path: "/path/to/ballerinaKeystore.p12",
             password: "ballerina"
         }
     }
 });
 
 @http:ServiceConfig {
-    basePath: "/hello",
-    auth: {
-        scopes: ["hello"]
-    }
+    auth: [
+        {
+            oauth2IntrospectionConfig: {
+                url: "https://localhost:9999/oauth2/token/introspect",
+                tokenTypeHint: "access_token",
+                scopeKey: "scp",
+                clientConfig: {
+                    secureSocket: {
+                        trustStore: {
+                            path: "/path/to/ballerinaTruststore.p12",
+                            password: "ballerina"
+                        }
+                    }
+                }
+            },
+            scopes: ["hello"]
+        }
+    ]
 }
-service helloWorld on secureHelloWorldEp {
-
-    @http:ResourceConfig {
-        methods: ["GET"],
-        path: "/"
-    }
-    resource function sayHello(http:Caller caller, http:Request req) {
-        http:Response resp = new;
-        resp.setTextPayload("Hello, World!");
-        checkpanic caller->respond(resp);
+service /foo on securedEP {
+    resource function get bar() returns string {
+        return "Hello, World!";
     }
 }
 ```
 
 When the service is invoked without authentication information or invalid authentication information, an authentication failure will occur:
 
 ```
-curl -k -v https://localhost:9091/hello
+curl -k -v https://localhost:9090/foo/bar
 
-> GET /hello HTTP/1.1
-> Host: localhost:9091
+> GET /foo/bar HTTP/1.1
+> Host: localhost:9090
 > User-Agent: curl/7.47.0
 > Accept: */*
 > 
 < HTTP/1.1 401 Unauthorized
+< content-length: 0
+< server: ballerina
 < content-type: text/plain
 < 
-Authentication failure
 ```
 
-If a request is made with valid authentication information, but the LDAP server responds with an empty group list or unexpected scopes, an authorization failure will occur.
+If a request is made with valid authentication information, but the authenticated user does not have the required permission, an authorization failure will occur.
 
 ```
-curl -k -v https://localhost:9091/hello -H "Authorization: Basic <token>"
+curl -k -v https://localhost:9090/foo/bar -H "Authorization: Bearer <token>"
 
-> GET /hello HTTP/1.1
-> Host: localhost:9091
+> GET /foo/bar HTTP/1.1
+> Host: localhost:9090
 > User-Agent: curl/7.47.0
 > Accept: */*
-> Authorization:Bearer <token>
+> Authorization: Bearer <token>
 >
 
 < HTTP/1.1 403 Forbidden
+< content-length: 0
+< server: ballerina
 < content-type: text/plain
 <
-Authorization failure
 ```
 
-A request, which gets a successful response from the LDAP server for the "scope" request will result in a successful invocation.
+A request is made with valid authentication information, and the authenticated user has the required permission, will result in a successful invocation.
```

Comment:
```suggestion
When a request is made with valid authentication information, if the authenticated user has the required permission, this will result in a successful invocation.
```